src/erlcloud_ecs.erl

```diff
@@ -1990,6 +1991,48 @@ list_services(Opts, Config) ->
     out(Return, fun(Json, UOpts) -> decode_record(list_services_record(), Json, UOpts) end,
         EcsOpts).
 
+%%%------------------------------------------------------------------------------
```

Comment:
It looks like the style is to have a section heading like this at the "top"

```erlang
%%%------------------------------------------------------------------------------
 %% ListTagsForResource
 %%%------------------------------------------------------------------------------
```

Then put this block with the full docs above the highest arity function, `list_tags_for_resource/3` here. I think that's the last change and G2G.